core/common/src/BackendTypes.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { Readable, Writable } from "stream"; // Must be "import type" to avoid webpack polyfill errors
+import type { Buffer } from "buffer"; // Must be "import type" to avoid webpack polyfill errors
+
+/*
+IMPORTANT:
+All types in this file will be removed in 4.0
```

Comment:
please make this say that this file is temporary for backwards compatibility and don't add new types to it.

Reply:
Will do.